MapboxNavigationTests/SimulatedLocationManagerTests.swift

```diff
@@ -13,7 +13,15 @@ class SimulatedLocationManagerTests: FBSnapshotTestCase {
     }
 
     func testSimulateRouteDoublesBack() {
-        let route = Fixture.routesFromMatches(at: "sthlm-double-back")![0]
+        let routeExpectation = expectation(description: "Gets a matching route")
+        var route: Route!
+        Fixture.getRoutesFromMatches(at: "sthlm-double-back") { (waypoints, routes, error) in
+            route = routes?[0]
+            routeExpectation.fulfill()
+        }
+        wait(for: [routeExpectation], timeout: 1)
+        guard route != nil else { return }
```

Comment:
Shouldn't we fail the test if this guard fails?

Reply:
As above, the test will fail due to an unfulfilled expectation. The guard is needed here to keep the test failure from becoming a test crash, which is more difficult to diagnose in CI logs.